pkg/options/options.go

```diff
@@ -62,7 +62,7 @@ type Options struct {
 	cmd *cobra.Command
 }
 
-// GetOptsConfigFile is the getter for --options-config-file value.
+// GetOptsConfigFile is the getter for --config value.
```

Comment:
@rexagod Let me know if you think we should rename the function as well here

Reply:
Yep, makes sense! 👍